content/en/docs/reference/tools/_index.md

```diff
@@ -1,8 +1,13 @@
 ---
+title: Tools
 reviewers:
 - janetkuo
-title: Tools
 content_type: concept
+weight: 80
+no_list: true
+card:
+  name: reference
```

Comment:
Hi @raghvenders . 

I don't know if you need the changes in this page?

The `card` field adds this page as a link (`Tools`) to the docs home page. Is this what you intend?

Reply:
The intention is to keep Tools as not selected link like other links but it appears otherwise.
You can verify it as in the left navigator Reference listing. , so I created and moved it under Tools Folder. 

The intention is not to move it to the Home page but to feel encouraged if it is agreeable and helps.

Reply:
Hi @raghvenders . Thanks for the updates. Here is some feedback:
- I see the new links added to the reference _index page. The order is slightly different from the left hand navigation tree.
- There is no longer a link to the `Tools` page. Do you want to keep a link to the Tools page on this _index page or leave this link off the page?
- Do you want to add the `Tools` link to the Docs home page?

You could remove the card data or leave the data to add the new `Tools` link,

```
card:
  name: reference
```

Reply:
Sure @kbhawkey, I will pull the master to have links update on top.